images/1try/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>twitter</title>
+
+    <link rel="stylesheet" href="Scontainer.css">
+    <link rel="stylesheet" href="left.css">
+
+</head>
+
+<body>
+
+    <!-- main layout -->
+    <div class="container">
+        <!-- leftmenu -->
+        <div class="leftmenu">
+            <ul class="twitter">
+                <li> <img src="https://img.icons8.com/wired/64/000000/branding.png" /></li>
+                <li><strong> <img src="https://img.icons8.com/wired/64/000000/hiragana-ma.png" />j zjjz</strong></li>
+                <li><strong> <img src="https://img.icons8.com/material/64/000000/hiragana-ma.png" /> Menub</strong></li>
+                <li><strong> <img src="https://img.icons8.com/wired/64/000000/beta.png" /> Menub</strong></li>
+                <li><strong><img src="https://img.icons8.com/wired/64/000000/alpha.png" /> Menub</strong></li>
+                <div class="endbotton"> soy un bottone</div>
+                </ulclass=>
```

Comment:
The `</ul>` is not written and closed correctly. You may want to remove that **ulclass** and replace it with `</ul>`.